BEACON-V2-Model/cohorts/endpoints.json

```diff
@@ -26,6 +26,8 @@
     "/cohorts": {
       "get": {
         "parameters": [
+          { "$ref": "#/components/parameters/ids" },
+          { "$ref": "#/components/parameters/datasetIds" },
```

Comment:
Cohorts are not part of datasets. Hence this parameter shouldn't be applied to this entry type.

Reply:
But there is an intersect; and since some data/conditions are very dataset specific (e.g. DACs are usually per dataset) so it seems a reasonable option (all under the assumption that we support cross-dataset cohorts, which IMO we do).. However, I'm not pushing this in; but then we need a documentation of the usage scenarios.

Again: I can remove this for now, but please confirm.